### PR TITLE
Mp code cleanup final

### PR DESCRIPTION
**Pull request by pattmax00**
A bunch of formatting I brought over from the other branch I had and I renamed the variables to match our coding standards. Everything still builds and runs without errors.

### CODE REVIEW

File: AcademicReward/Database/AcademicRewardsDatabase.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public class AcademicRewardsDatabase {
     private const string BitApiKey = "v2_3vaWv_9N4SgRBSZweqRCh5mKAQ4BE";
     private const string BitUsername = "pattmax";
     private const string BitDatabaseName = "pattmax/AcademicRewards";
-    private string connectionString = string.Empty;
+    private string _connectionString = string.Empty;
```
Comment (Wil-LaLonde) [inline AcademicReward/Database/AcademicRewardsDatabase.cs:16]: Were these underscores intended? Should these variable names really have an underscore at the start? I see quite a few of them regarding our database stuff as well. Some of our property values have it too

Comment (pattmax00) [reply AcademicReward/Database/AcademicRewardsDatabase.cs:16]: Yes, local private variables should be prefixed with an underscore

Comment (pattmax00) [reply AcademicReward/Database/AcademicRewardsDatabase.cs:16]: https://stackoverflow.com/questions/3136594/naming-convention-underscore-in-c-and-c-sharp-variables

Comment (Wil-LaLonde) [reply AcademicReward/Database/AcademicRewardsDatabase.cs:16]: I had no idea, looks good to me

**Review by Wil-LaLonde — APPROVED** [review]

**Pull request merged**